### PR TITLE
add node cli subcommand

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

**Comment by tamarrow** [comment]
@jsancio bump

File: repo/packages/N/node/0/command.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "pip": [
+    "dcos<1.0",
+    "git+https://github.com/mesosphere/dcos-node.git#dcos-node=0.1.0"
```
Comment (jsancio) [inline repo/packages/N/node/0/command.json]: Let's install from PyPI.

Comment (jsancio) [inline repo/packages/N/node/0/command.json]: And this won't work for any of our users because that URL is private.

**Comment by kensipe** [comment]
@tamarrow what is our plan with this?  the CLI subcommand project repo is private.. we can't merge this that way.

**Comment by tamarrow** [comment]
We decided to punt on this until we decide what we want out of it.

**Comment by jsancio** [comment]
Closing for now.

**Pull request closed without merging**